forms.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask
+from flask_wtf import FlaskForm, CSRFProtect
+from wtforms import StringField, PasswordField, SubmitField
+from wtforms.validators import DataRequired, Email, Length, EqualTo
+
+from main import app
+
+app.config['SECRET_KEY'] = b''
+csrf  = CSRFProtect(app)
+
+class RegistrationForm(FlaskForm):
+    username = StringField('Name', validators=[DataRequired()])
+    surname = StringField('Surname', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
```

Comment:
Можно использовать поле `EmailField`
```suggestion
    email = EmailField('Email', validators=[DataRequired(), Email()])
```